tutordistro/plugin.py

```diff
@@ -121,7 +121,7 @@
             "index": "git",
             "name": "eox-audit-model",
             "repo": "eox-audit-model",
-            "version": "v1.0.0",
+            "version": "v2.0.0",
             "domain": "github.com",
             "protocol": "https",
             "path": "eduNEXT",
```

Comment:
We need to change nuez tag by olmo tag in this file (ex: the theme and the docker overrides)

Reply:
on it! thanks